ddp-workspace/angular.json

```diff
@@ -1987,7 +1987,7 @@
               "projects/ddp-rgp/src/assets",
               {
                 "glob": "*.pdf",
-                "input": "projects/ddp-rgp/src/pdf/",
+                "input": "projects/ddp-rgp/src/assets/pdf/",
```

Comment:
Looks like this is looking at the wrong folder?

Reply:
Right, now it's correct